pkg/hintrunner/zero/zerohint_ec.go

```diff
@@ -538,6 +531,52 @@ func createEcDoubleAssignNewXV1Hinter(resolver hintReferenceResolver) (hinter.Hi
 	return newEcDoubleAssignNewXV1Hint(slope, point), nil
 }
 
+// EcDoubleAssignNewYV1 hint computes a new y-coordinate when doubling a point
+// on an elliptic curve
+//
+// `newEcDoubleAssignNewYV1Hint` doesn't take any operander as argument
+// This hint follows the execution of `EcDoubleAssignNewXV1` hint when doubling a point
+// This is why all variables are already accessible in the current scope
```

Comment:
... this description different if the code performs the same operations

Reply:
What do you mean ? They borrow the same logique but one does addition of points , the other double the same point ?